### PR TITLE
[16.0][ADD] product_variant_name

### PR DESCRIPTION
**Pull request by JordiMForgeFlow**
Allow modifying the product variant name.

### CODE REVIEW

File: product_variant_name/init_hook.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+def post_init_hook(cr, registry):
+    """Init product variant name with product template name"""
+    logger.info("Setting product variant name with product template name")
+    # FOR stock moves
```
Comment (LoisRForgeFlow) [inline product_variant_name/init_hook.py]: stock moves?

**Review by LoisRForgeFlow — APPROVED** [review]
Code review LGTM

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM

**Review by AaronHForgeFlow — APPROVED** [review]
Functional review. It works!

**Pull request merged**